openpype/modules/kitsu/plugins/publish/integrate_kitsu_note.py

```diff
@@ -19,19 +22,29 @@ def process(self, context):
 
         self.log.debug("Comment is `{}`".format(publish_comment))
 
-        # Get Waiting for Approval status
-        kitsu_status = gazu.task.get_task_status_by_short_name("wfa")
-        if not kitsu_status:
-            self.log.info(
-                "Cannot find 'Waiting For Approval' status."
-                "The status will not be changed"
+        # Get note status, by default uses the task status for the note 
+        # if it is not specified in the configuration
+        note_status = context.data["kitsu_task"]["task_status_id"]
+        if self.set_status_note:
+            kitsu_status = gazu.task.get_task_status_by_short_name(
+                self.note_status_shortname
             )
-            kitsu_status = context.data["kitsu_task"].get("task_status")
-        self.log.debug("Kitsu status: {}".format(kitsu_status))
+            if not kitsu_status:
+                self.log.info(
+                    "Cannot find {} status. The status will not be "
+                    "changed!".format(self.note_status_shortname)
+                )
+            else:
+                note_status = kitsu_status
+                self.log.info("Note Kitsu status: {}".format(note_status))
```

Comment:
It's clearer to write things in a positive way.
```suggestion
            if kitsu_status:
                note_status = kitsu_status
                self.log.info("Note Kitsu status: {}".format(note_status))
            else:
                self.log.info(
                    "Cannot find {} status. The status will not be "
                    "changed!".format(self.note_status_shortname)
                )
```